cypress/integration/inventory/update-effective-location-for-item.spec.js

```diff
@@ -9,14 +9,14 @@ import getRandomPostfix from '../../support/utils/stringTools';
 import generateItemBarcode from '../../support/utils/generateItemBarcode';
 import permissions from '../../support/dictionary/permissions';
 import Users from '../../support/fragments/users/users';
-import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
 
-describe('Update the effective location for the item', () => {
+describe('ui-inventory: Update the effective location for the item', () => {
   const instanceTitle = `autoTestInstanceTitle.${getRandomPostfix()}`;
   const anotherPermanentLocation = 'Main Library';
   const ITEM_BARCODE = generateItemBarcode();
```

Comment:
Why is the name of this variable ITEM_BARCODE fully uppercase?

Reply:
I don't know

Reply:
It is better to have the same formatting to all similar elements. So please rename it to itemBarcode because you use CamelCase and write the variables in lowercase letters

Reply:
Done